analyzers/src/SonarAnalyzer.Utilities/RuleDescriptors/RuleDetail.cs

```diff
@@ -22,15 +22,15 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Reflection;
 using System.Resources;
+using Microsoft.CodeAnalysis;
 using SonarAnalyzer.Common;
 
 namespace SonarAnalyzer.RuleDescriptors
 {
     public sealed class RuleDetail
     {
-        private static readonly Assembly SonarAnalyzerUtilitiesAssembly = typeof(RuleDetail).Assembly;
+        private static readonly string RspecRoot = Path.Combine(Path.GetDirectoryName(typeof(RuleDetail).Assembly.Location), $@"..\..\..\..\..\rspec");
```

Comment:
First half of my best line, extracted for performance and readability

Reply:
It would be useful to add here a comment with relative location from the repository. In this way, we will not have to count folders when reading this, and also, when debugging in case of problems we will see that the expectation is met.